app/views/hub/bulk_message_csvs/index.html.erb

```diff
@@ -2,23 +2,7 @@
   <div class="slab">
     <div class="grid">
       <h1><%= @main_heading %></h1>
-      <p>You can use this to send bulk messages. You should use this with great care;
-        coordinate with the program team. Here are the steps.</p>
-
-      <ul class="list--padded list--bulleted">
-        <li>Upload a CSV on this page (1 column, just client_id, header row is required and must be "client_id",
-          filename is displayed on this page).
-        </li>
-        <li>Reload this page until you see the words "Send a message" next to your CSV filename.</li>
-        <li>Click on the contact method(s) you want to use.</li>
-        <li>Type the English & Spanish into the box, then click Submit.
-        <li>You will now see your notifications page ("My Updates"). The messages will start sending. You can reload
-          that page to check on the status.
-        </li>
-      </ul>
-
-      <p>This tool will send messages at whatever ungodly hour of the day you ask it to; if you want to make sure that
-        your messages don’t go out at midnight, it’s on you. The tool may send about 1 message per second.</p>
+      <p> <a href="https://www.notion.so/cfa/Bulk-Messaging-Read-Me-6559552f524340569c11583186d76bda"> Click here for instructions and important warnings on how to use this page </a></p>
```

Comment:
I would add `target="_blank" rel="noopener noreferrer"` into the a tag so that it looks like:
```
<p> <a href="https://www.notion.so/cfa/Bulk-Messaging-Read-Me-6559552f524340569c11583186d76bda" target="_blank" rel="noopener noreferrer"> Click here for instructions and important warnings on how to use this page </a></p>
```